logger/logger_file.go

```diff
@@ -0,0 +1,40 @@
+package logger
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+type fileWriters struct {
+	fileHandle []os.File
+}
+
+var f = fileWriters{}
+
+func createFileHandle(subsystem string) (io.Writer, error) {
```

Comment:
Perhaps make this ability to write to custom subsystems an advanced setting as I tend to like all logs in one single file, even though others might prefer a log file per subsystem

Reply:
Made the change to support both

there is "file" and "ownfile" output options now
with file being a global file handle that writes to GCTPATH/logs/log.txt and ownfile being its own that writes to GCTPATH/logs/<loggername.txt> (you can also write to both at the same time if you wish)